PowerEditor/src/langs.model.xml

```diff
@@ -72,7 +72,7 @@
         </Language>
         <Language name="batch" ext="bat cmd nt" commentLine="REM">
-            <Keywords name="instre1">assoc aux break call cd chcp chdir choice cls cmdextversion color com com1 com2 com3 com4 con copy country ctty date defined del dir do dpath echo else endlocal erase errorlevel exist exit for ftype goto if in loadfix loadhigh lpt lpt1 lpt2 lpt3 lpt4 md mkdir move not nul path pause popd prn prompt pushd rd rem ren rename rmdir set setlocal shift start time title type ver verify vol</Keywords>
+            <Keywords name="instre1">assoc aux break call cd chdir cls cmdextversion color com com1 com2 com3 com4 con copy country ctty date defined del dir do dpath echo else endlocal erase errorlevel exist exit for ftype goto if in loadfix loadhigh lpt lpt1 lpt2 lpt3 lpt4 md mkdir move not nul path pause popd prn prompt pushd rd rem ren rename rmdir set setlocal shift start time title type ver verify vo</Keywords>

```

Comment:
`vo` is not recognized as an internal or external command... `vol` is a known command. Seems like a typo. Can you fix? Other then that minor issue, batch keywords looks reasonably good to me.